src/server/editorServices.ts

```diff
@@ -2764,6 +2775,10 @@ namespace ts.server {
                     this.hostConfiguration.watchOptions = convertWatchOptions(args.watchOptions);
                     this.logger.info(`Host watch options changed to ${JSON.stringify(this.hostConfiguration.watchOptions)}, it will be take effect for next watches.`);
                 }
+
+                if(args.typeAcquisition) {
+                    this.hostConfiguration.typeAcquisition = args.typeAcquisition;

```

Comment:
There is no need for hostConfiguration as there  are no settings that are not applied to projects.. Eg. watchOptions needed host configuration because we watch things project service wide rather than per project.,